x-pack/plugins/index_lifecycle_management/public/store/reducers/nodes.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { handleActions } from 'redux-actions';
+import {
+  fetchedNodes,
+  setSelectedNodeAttrs,
+  setSelectedPrimaryShardCount,
+  setSelectedReplicaCount,
+  fetchedNodeDetails
+} from '../actions/nodes';
+
+const defaultState = {
+  isLoading: false,
+  selectedNodeAttrs: '',
+  selectedPrimaryShardCount: 1,
+  selectedReplicaCount: 1,
+  nodes: [],
```

Comment:
Changing this to `nodes: undefined` will solve the problem.

Reply:
good catch thx